setup.py

```diff
@@ -15,7 +16,9 @@
     ],
     tests_require=[
         'mock>=1.0.1',
+        'flake8',
     ],
+    test_suite='nose.collector',
```

Comment:
Hmm. The "find the tests" part sounds compelling. I've been running tests with something like `for x in *_test.py; do python $x; done`.